beets/util/functemplate.py

```diff
@@ -318,9 +318,11 @@ def __init__(self, string):
 
     # Common parsing resources.
     special_chars = (SYMBOL_DELIM, FUNC_DELIM, GROUP_OPEN, GROUP_CLOSE,
-                     ARG_SEP, ESCAPE_CHAR)
+                     ESCAPE_CHAR)
     special_char_re = re.compile(r'[%s]|$' %
                                  u''.join(re.escape(c) for c in special_chars))
+    escapable_chars = (SYMBOL_DELIM, FUNC_DELIM, GROUP_CLOSE, ARG_SEP)
```

Comment:
Yes, I agree here—I think it's more predictable to make $, work the same way—as an escape—both inside and outside of function arguments.